etc/kayobe/ansible/upload-database-backup-s3.yml

```diff
@@ -0,0 +1,35 @@
+---
```

Comment:
This could use some documentation, probably in somewhere like `doc/source/operations/database-backups.rst`